README.md

```diff
@@ -102,6 +102,12 @@ At the moment, MBARI WEC is supported by source installation only. Use Ubuntu Ja
     export GZ_VERSION=garden
     ```
 
+1. Set `SDF_PATH` to allow `robot_state_publisher` parse the robot description from the sdformat model:
+
+   ```
+   export SDF_PATH=$GZ_SIM_RESOURCE_PATH
```

Comment:
Hmm, I think `$GZ_SIM_RESOURCE_PATH` is blank at this point. If they have no other Gazebo workspaces, they have to compile the workspace and run `install/setup.bash` before `$GZ_SIM_RESOURCE_PATH` has any value. So this line should be moved to the Run section, after they've sourced `install/setup.sh`.

Actually, I have a question about the Run section, why do we choose to use `setup.sh` as opposed to `setup.bash` there? Could the user be using something that's not bash? We are already asking them to source `/opt/ros/humble/setup.bash`, which assumes bash shell. I think it should be consistent across the document whether we're using .sh or .bash.

Reply:
Yeah could you make a pass over the docs to make everything use `setup.bash`?

Reply:
Done!